modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -4,7 +4,10 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: esb, deploy, configuration, cluster, ha, high availability, downtime
 
-A *cluster* is a set of Mule runtimes that acts as a unit. In other words, a cluster is a virtual server composed of multiple nodes. The nodes (Mule runtimes) in a cluster communicate and share information through a distributed shared memory grid. This means that the data is replicated across memory in different physical machines.
+[IMPORTANT]
+Clustering is not available in CloudHub, see xref:runtime-manager::cloudhub-fabric.adoc[CloudHub Fabric] for details on how workers can be shared or doubled to scale your application and provide high availability.
+
+A Cluster is a set of Mule runtime engines that acts as a unit. In other words, a cluster is a virtual server composed of multiple nodes. The nodes (Mule runtime engines) in a cluster communicate and share information through a distributed shared memory grid. This means that the data is replicated across memory in different machines.
```

Comment:
I suggest putting (Mule runtime engines) in the first sentence, after the word "nodes." It breaks up the second sentence., Also, Cluster should be lowercase.